src/ontology/genepio-edit.owl

```diff
@@ -4234,7 +4234,7 @@ SubClassOf(obo:GENEPIO_0000008 obo:GENEPIO_0002075)
 # Class: obo:GENEPIO_0000009 (other (metadata value))
 
 AnnotationAssertion(obo:IAO_0000114 obo:GENEPIO_0000009 obo:IAO_0000428)
-AnnotationAssertion(obo:IAO_0000115 obo:GENEPIO_0000009 "\"other (metadata choice)\" indicates that for the given categorical variable, a respondent has chosen an other or additional response not listed."@en)
+AnnotationAssertion(obo:IAO_0000115 obo:GENEPIO_0000009 "A data item \\\"other (metadata choice)\\\" that indicates for the given categorical variable, a respondent has chosen an other or additional response not listed."@en)
```

Comment:
Its not really needed to repeat the "other (metadata choice)" data item label in the definition."

Reply:
Thanks, will amend in following commit.